OSVR-Unity/Assets/OSVRUnity/src/Math.cs

```diff
@@ -45,6 +45,11 @@ public static Quaternion ConvertOrientation(OSVR.ClientKit.Quaternion quat)
                 /// Wikipedia may say quaternions are not handed, but these needed modification.
                 return new Quaternion(-(float)quat.x, -(float)quat.y, (float)quat.z, (float)quat.w);
             }
+            public static Quaternion ConvertOrientationFromRenderManager(OSVR.ClientKit.Quaternion quat)
```

Comment:
I didn't know that they were.  With RenderManager, you can add a room-to-world transform and make them whatever you want, but that has now also been added to Core with the reset-yaw code.

The Core transforms and RenderManager transforms were developed independently, so may have different coordinate systems.  But OSVR defines a standard, so if they are different then one of them is wrong.